src/data/data.js

```diff
@@ -32,19 +32,24 @@ class Data extends Base {
     super(...params);
 
     /** @type {Metrics} */
-    this.metrics = new Metrics(this);
+    this.Metrics = new Metrics(this);
+    this.metrics = this.Metrics;
```

Comment:
Why do you still need these? Just so this doesn't break backwards compatibility?

Reply:
Yeah, just so there'd be one less big change to make if you were updating from 1.x (you already have to remove all the `data.data` accessors)